splunkforwarder/elx/init.sls

```diff
@@ -96,7 +96,7 @@ Configure Local Log Sources:
 
 Accept Splunk License:
   cmd.run:
-    - name: {{ splunkforwarder.bin_file }} start --accept-license
+    - name: {{ splunkforwarder.bin_file }} start --accept-license --answer-yes --no-prompt
```

Comment:
Does this require a minimum version of the splunk-forwarder to work? If so, perhaps it would be better to expose a new pillar option that is passed through to the start command? Similar to the `installer_opts` option in the [mcafee agent formula](https://github.com/plus3it/mcafee-agent-formula/blob/e5ee0801f300a8aa2da645a2e5baf09f78458033/mcafee-agent/elx/init.sls#L66)

Reply:
I verified that it still works with 7.0.2 but haven't checked with older versions.  I'll look into adding the installer options.